### DISCUSSION TITLE
How to have client id in AdGuardHome

### DISCUSSION THREAD

**Discussion by multinet33** [AdguardTeam/AdGuardHome#6836] [Q&A]
Hello 

I've put my adguardhome IP  (on a raspi) in DNS section of my ORBI RBR 850 but in the dashboard all traffic is on the ip of the router

I've seen that I my enable port forwarding  but i'm a bit confuse.

Where to enable it ? on my orbi routeur ? 
what is the correct forwarding rule ? ext IP udp 53 to adguardhome IP port 53 ?

thanks for help

**Comment by ghost** [comment]
Try using AdGuard Home as the [DHCP server](https://github.com/AdguardTeam/AdGuardHome/wiki/DHCP) which should resolve this. Not having an ORBI I can't specifically test this.

**Reply by multinet33** [reply]
Thanks for your reply, 

I don't find the way to configure a dhcpd on adguardhome (assigna specif IP for a specif mac adress) ?
